.github/workflows/nightly.yml

```diff
@@ -70,3 +70,114 @@ jobs:
           type: "stream"
           topic: "CI - Doctrine/DBAL"
           content: "There was an error running Doctrine on MariaDB:${{ matrix.mariadb-version }} - URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}."
+
+  phpunit-sqlite:
+    name: PHPUnit with SQLite
+    uses: ./.github/workflows/phpunit-sqlite.yml
+    with:
+      os: ${{ matrix.os }}
+      php-version: ${{ matrix.php-version }}
+      extension: ${{ matrix.extension }}
+      dependency-versions: ${{ matrix.dependency-versions }}
+
+    strategy:
+      matrix:
+        os:
+          - ubuntu-24.04
+        php-version:
+          - '8.5'
```

Comment:
should this use the `nightly` keyword to reduce maintenance work for that workflow ?

Reply:
If that's the same, yes.

Reply:
the difference is that `nightly` will automatically mean PHP 8.6 after the release of PHP 8.5.0 (it always refers to the `master` branch of PHP).

Actually, `8.5` currently installs the nightly PHP version as 8.5 is not released yet: https://github.com/shivammathur/setup-php?tab=readme-ov-file#tada-php-support

to me, `nightly` is exactly the intent we want in this `nightly.yml` job, to avoid having to remember to update it every year.

Reply:
Yeah, that was my understanding. Using `nightly` sounds good. It's more expressive and requires less maintenance.